LayoutTests/compositing/visibility/omitted-nested-hidden-layers-crash.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ AsyncOverflowScrollingEnabled=true ] -->
+<html lang="en">
+<head>
+<style>
+    .header {
+        padding: .1rem .1rem;
+        position: relative;
+        width: 100%;
+    }
+
+    li {
+        position: relative;
+        line-height: 100rem;
+    }
+
+    .overlap {
+        height: 70vh;
+        margin-top: -.01rem;
+        max-height: 200px;
+        overflow-x: hidden;
+        overflow-y: auto;
+        width: 100%;
+    }
+
+    .aaaa {
+        max-width: 300px;
+        max-height: 200px;
+    }
+</style>
+</head>
+<body>
+
+<p>This test should not crash.</p>
+
+<div id="wrapper">
+    <div class="aaaa">
+        <div class="header">
+        </div>
+        <div class="overlap">
+            <ul>
+                <li></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+<script>
+if (window.testRunner) {
+    testRunner.waitUntilDone();
+    testRunner.dumpAsText();
+}
+
+window.onload = async () => {
+    await new Promise(requestAnimationFrame);
+    document.getElementById("wrapper").style.visibility = "hidden";
+    
+    await new Promise(requestAnimationFrame);
```

Comment:
Do you really want to `await UIHelper.renderingUpdate()` ? `await new Promise(requestAnimationFrame);` resumes when the rAF callback returns, before the end of `Page::updateRendering()` and any transaction building.

Reply:
I think we flush layout for the load event, so it works either way, but probably better with `UIHelper.renderingUpdate()`.